crates/winscard/src/scard.rs

```diff
@@ -362,7 +362,7 @@ impl SmartCard<'_> {
         // NIST.SP.800-73-4, Part 1, Table 5
         const RSA_ALGORITHM: u8 = 0x07;
         // NIST.SP.800-73-4, Part 1, Table 4b
-        const PIV_DIGITAL_SIGNATURE_KEY: u8 = 0x9C;
+        const PIV_DIGITAL_SIGNATURE_KEY: u8 = 0x9A;
```

Comment:
The value of this constant is correct. The scard auth using `mstsc.exe` will break if we change it. I suppose the name is bad. I think we can rename it, but we can't change its value